### ISSUE TITLE
mention missing packages

### ISSUE THREAD

**Issue by peterducai** [github/choosealicense.com#693]
please mention requirements before running bootstrap... on my Fedora 31 i had to install

> sudo dnf install zlib ruby-devel zlib-devel

thanks

**Comment by peterducai** [comment]
also missing gems

```
gem install commonmarker -v '0.17.13' --source 'https://rubygems.org/'
gem install commonmarker -v '0.17.13' --source 'https://rubygems.org/'
gem install nokogiri -v '1.10.4' --source 'https://rubygems.org/'
sudo gem pristine eventmachine --version 1.2.7
sudo gem pristine ffi --version 1.11.1
gem install github-pages
gem pristine http_parser.rb --version 0.6.0
sudo gem pristine http_parser.rb --version 0.6.0
gem install github-pages
gem install html-proofer
gem install licensee
```

**Comment by peterducai** [comment]
but even after that i'm stuck with licensee gem on

```
find_executable: checking for gmake... -------------------- yes

--------------------

find_executable: checking for cmake... -------------------- yes

--------------------

find_executable: checking for pkg-config... -------------------- yes

--------------------

/usr/bin/gmake
gmake: *** No targets specified and no makefile found.  Stop.
```

**Comment by nschonni** [comment]
> please mention requirements before running bootstrap... on my Fedora 31 i had to install
> 
> > sudo dnf install zlib ruby-devel zlib-devel

That is a little buried but is a general Jekyll thing https://jekyllrb.com/docs/installation/other-linux/

A bunch of those look like they should have been covered by the Gemfile https://github.com/github/choosealicense.com/blob/gh-pages/Gemfile

**Comment by mlinksva** [comment]
`script/bootstrap` as stated in https://github.com/github/choosealicense.com/tree/8a290cffad311dbb45db1074a0a42ffecfb2964f#run-it-on-your-machine should install the needed gems.

I don't think detailed platform specific requirements should be spelled out in this repo but I can imagine a link to https://jekyllrb.com/docs/installation/ could be helpful; feel free to make a PR.

**Comment by mlinksva** [comment]
I think this was addressed by #765. If further documentation is needed, feel free to make a PR.
